functions/src/post-document-comment.ts

```diff
@@ -28,7 +28,7 @@ export async function postDocumentComment(
     throw new functions.https.HttpsError("invalid-argument", "Some required document information was not provided.");
   }
 
-  if (!comment?.content) {
+  if (!comment?.content && !comment?.tags?.length) { //if we have a tag we can post an empty comment
```

Comment:
This is technically part of this ticket
https://www.pivotaltracker.com/n/projects/2441242/stories/185633009
But I'll leave it for now.